src/core/Game.ts

```diff
@@ -137,7 +137,7 @@ export class Game {
     private _animate(): void {
         this._render();
         Game.deltaTime = View.clock.getDelta();
-        Controls.makeObjectMove({obj: this.controls.getCamera(), diffPosition: Controls.cameraSpeed})
+        this.controls.makeObjectMove({obj: this.controls.getCamera(), diffPosition: this.controls.cameraSpeed})
```

Comment:
create a method on `Controls` class (getter) for `this.controls.cameraSpeed` called `getCameraSpeed()`, and access `Controls.cameraSpeed` value from there